Jenkinsfile

```diff
@@ -104,11 +113,13 @@ pipeline {
                     }
                 }
                 stage('rudder-pkg') {
-                    agent { label 'script' }
+                    agent { docker { image 'python'
+                                     args '-v /etc/passwd:/etc/passwd:ro --tmpfs /srv/jenkins/.local:exec' } }
+
                     steps {
                         dir ('relay/sources') {
-                            sh script: 'typos --exclude "api-doc/*" --exclude "relayd/*"', label: 'check typos'
-                            sh script: 'make check', label: 'rudder-pkg tests'
+                            sh script: 'pip install -r rudder-pkg/requirements.txt', label: 'install dependencies'
```

Comment:
Shouldn't the dependencies be installed at image build time?

Reply:
May be better indeed.